src/PerformanceTests/readme.md

```diff
@@ -22,10 +22,18 @@ Table of contents
 
 
 
-# Prerequisites for running tests
+# Prerequisites
 
-- Making sure that connection string to MS SQL DB is correct
-- Creating empty database manually before first run
+Tools for development and testing:
+
+- Visual Studio 2017 (tested with 15.3.5)
+- [Slow Cheetah for vs2017](https://marketplace.visualstudio.com/items?itemName=VisualStudioProductTeam.SlowCheetah-XMLTransforms)
```

Comment:
is this really required anymore? I see a Microsoft-vs-slow-cheetah nuget package on some projects but that doesn't seem to require the slow cheetah plugin?

Reply:
The extension is required in order to view and test the transformations within the v2017 UI but is not needed to build the projects.